src/Checkbox/Checkbox.styles.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { CheckboxProps } from './types';
+import { getBackgroundColorFromState, getBorderColorFromState, getColorFromState } from './utils';
+
+interface StyledCheckboxProps extends CheckboxProps {
+  $indeterminate?: boolean;
+}
+
+export const StyledCheckbox = styled.input<StyledCheckboxProps>`
+  appearance: none;
+  border-radius: 2px;
+  border: 1px solid ${getBorderColorFromState};
+  cursor: pointer;
```

Comment:
Maybe show default cursor instead of pointer when toggle is disabled:

```
cursor: ${({ $disabled }) => ($disabled ? 'default' : 'pointer')};
```

Reply:
Good catch. Fixing it.

Reply:
Fixed.